### ISSUE TITLE
Dev tools should not open in a new window

### ISSUE THREAD

**Issue by Jaiganeshkumaran** [FireCubeStudios/SwiftBrowser#46]
Dev tools should open as sidebar instead of a separate window.

**Comment by FireCubeStudios** [comment]
dev tools currently only show an update the page html. i dont think it would be usefull in the side as that just take sup unnecessary space. ofcourse this is just my personal opinion

**Comment by Jaiganeshkumaran** [comment]
> dev tools currently only show an update the page html. i dont think it would be usefull in the side as that just take sup unnecessary space. ofcourse this is just my personal opinion

Also instead of having a update html button why can't you automatically refresh the page when the HTML is edited.

**Comment by FireCubeStudios** [comment]
yeah thats a good idea. because it using Monaco editor i need to know when the text has been changed.

**Comment by jaigak** [comment]
> yeah thats a good idea. because it using Monaco editor i need to know when the text has been changed.

Monaco Editor does provide a way to listen to changes however the UWP component doesn't expose it. If performance isn't that big of a concern, you may try using a periodic timer that runs every 500ms (or lower for better latency) to refresh.
